srcpkgs/hydrus/files/hydrus-server

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec python -OO /usr/lib/hydrus/server.py "$@"
```

Comment:
-||- and I can't see server.py in the package on x86_64, so this shouldn't work

Reply:
```
/usr/lib/hydrus/hydrus/test/TestServerDB.py
/usr/lib/hydrus/hydrus/test/__init__.py
/usr/lib/hydrus/server.py
/usr/lib/hydrus/static/8chan.png
/usr/lib/hydrus/static/Icons.svg
```

server.py is in the package and executable
should've been in there ever since, dunno what mightve happened to previous versions